models.go

```diff
@@ -307,24 +307,24 @@ type TaggedData struct {
 
 // Media ...
 type Media struct {
-	AspectRatio           string `json:"aspectRatio"`
-	AudioChannels         string `json:"audioChannels"`
-	AudioCodec            string `json:"audioCodec"`
-	AudioProfile          string `json:"audioProfile"`
-	Bitrate               string `json:"bitrate"`
-	Container             string `json:"container"`
-	Duration              string `json:"duration"`
-	Has64bitOffsets       string `json:"has64bitOffsets"`
-	Height                string `json:"height"`
-	ID                    string `json:"id"`
-	OptimizedForStreaming string `json:"optimizedForStreaming"`
-	Part                  []Part `json:"Part"`
-	Selected              bool   `json:"selected"`
-	VideoCodec            string `json:"videoCodec"`
-	VideoFrameRate        string `json:"videoFrameRate"`
-	VideoProfile          string `json:"videoProfile"`
-	VideoResolution       string `json:"videoResolution"`
-	Width                 string `json:"width"`
+	AspectRatio           float32 `json:"aspectRatio"`
+	AudioChannels         int     `json:"audioChannels"`
+	AudioCodec            string  `json:"audioCodec"`
+	AudioProfile          string  `json:"audioProfile"`
+	Bitrate               int     `json:"bitrate"`
+	Container             string  `json:"container"`
+	Duration              int     `json:"duration"`
+	Has64bitOffsets       bool    `json:"has64bitOffsets"`
+	Height                int     `json:"height"`
+	ID                    int     `json:"id"`
+	OptimizedForStreaming int     `json:"optimizedForStreaming"`
```

Comment:
`OptimizedForStreaming` is listed as a bool under the struct `Part`. Are they supposed to be different types?

Thanks for the contribution!

Reply:
Ah I think int is more appropriate, it's only a 1 or 0 but go will interpret it for use to a bool if we want. It really is an int though in the response. Under current sessions video it's a string. I don't use that struct but I just tested it on my own and it needs a re haul as well. The AddedAt is now a int as well and I imagine some of the similar fields that I changed here will need to be changed.

If you don't mind, I would love to try to consolidate some of the structs. So there would be one media struct that is used wherever a media field is found. Sometimes it looks like the structs are slightly different depending on the calling code which makes it a little harder to do.

Reply:
What kind of do you think you want with the minor differences? My go is a little rusty but would something like 

```
type MediaShared struct {
	AspectRatio           float32 `json:"aspectRatio"`
	AudioChannels         int     `json:"audioChannels"`
	AudioCodec            string  `json:"audioCodec"`
	AudioProfile          string  `json:"audioProfile"`
	Bitrate               int     `json:"bitrate"`
.......
	VideoFrameRate        string  `json:"videoFrameRate"`
	VideoProfile          string  `json:"videoProfile"`
	VideoResolution       string  `json:"videoResolution"`
	Width                 int     `json:"width"`
}
type MediaOther struct {
      MediaShared
      NewField int
}

type MediaOther2 struct {
      MediaShared
      OtherNewField int
}
```

Be reasonable? Although it doesn't seem like they usually switch types, it's just sometime the struct has more fields depending on the response. So if there is one Media struct that had everything, it could work, just some values would be unset/default depending on the request.